content/en/docs/reference/glossary/rkt.md

```diff
@@ -0,0 +1,18 @@
+---
+title: rkt
+id: rkt
+date: 2019-01-24
+full_link: https://coreos.com/rkt/
+short_description: >
+  A security-minded, standards-based container engine.
+
+aka: 
+tags:
+- security
+- tool
+---
+ A security-minded, standards-based container engine.
+
+<!--more--> 
+
+rkt is an application {% glossary_tooltip term_id="container" %} engine featuring a {% glossary_tooltip term_id="pod" %}-native approach, a pluggable execution environment, and a well-defined surface area. rkt allows users to apply different configurations  at both the pod and application level and each pod executes directly in the classic Unix process model, in a self-contained, isolated environment.
```

Comment:
The shortcodes should be in the form `{{< glossary_tooltip text="pod" term_id="pod" >}}`. Can you make that change? Apart from that, lgtm.